cloud/create-compute-instance-for-deployment-server.yml

```diff
@@ -0,0 +1,20 @@
+- hosts:
+    - localhost
+  vars_files:
+    - ../concent-builder/consts.yml
+    - "{{ deployment_values }}/var.yml"
+  vars:
+    server_name: concent-deployment-server
+    name_of_ip: "{{ server_name }}-ip"
+  roles:
+    - reserve_static_ip
+  tasks:
+    - block:
+        - name:    Machine running "{{ server_name }}"
+          command: >
+              gcloud compute instances create {{ server_name }}
+              --description  "Vm instance that contain {{ server_name }}"
+              --machine-type f1-micro
```

Comment:
Can we put this in `concent-deployment-values` ?

Reply:
Why do you want move it? The `cloud/create-compute-instances-for-geth.yml` playbook that creates geth instances is already in the `concent-deployment` repository.

Reply:
I think I complained about it back then too but I let you leave it as is if it turned out that moving it required significant effort.

Simply put, this is part of configuration. It should not be hard-coded here. It's not a very big deal, especially if moving it is a problem, but still, it does not belong here.